mkdocs.yml

```diff
@@ -1,8 +1,9 @@
 site_name: Snowfakery documentation
 site_url: ""
 use_directory_urls: False
-theme: readthedocs
-custom_dir: custom_theme
+theme:
+  name: readthedocs
+  custom_dir: custom_theme
```

Comment:
`ERROR   -  Config value 'theme': The path set in custom_dir ('custom_theme') does not exist.`

```suggestion
```